Common/TableProducer/trackPropagation.cxx

```diff
@@ -133,7 +133,20 @@ struct TrackPropagation {
           break;
       }
 
-      trackTunerObj.getDcaGraphs();
+      /// see if other track-propagation devices are present
+      int counter = 0;
+      LOG(info) << "  ---> looking for track-propagation devices";
+      const auto& workflows = initContext.services().get<RunningWorkflowInfo const>();
+      for (const DeviceSpec& device : workflows.devices) {
+        if (device.name.compare("track-propagation") == 0) {
+          // init HF event selection helper
+          LOG(info) << "    FOUND!";
```

Comment:
Thanks @mfaggin ! I would remove this log with "FOUND" in favor of something after the loop reporting the count of devices, i.e. `LOGF(info, "Number of track propagation tasks detected: %i", counter);` or so. Could you please adjust?